charts/gha-runner-scale-set/templates/autoscalingrunnerset.yaml

```diff
@@ -88,6 +88,11 @@ spec:
   minRunners: {{ .Values.minRunners | int }}
   {{- end }}
 
+  {{- with .Values.listenerTemplate}}
+  listenerTemplate:
+    {{- toYaml . | nindent 4}}
+  {{- end }}
```

Comment:
How is this different than `template` below it? And is there a valid reason why we aren't restrictive in the configuration we allow?

Reply:
I mean it is completely different field on the AutoscalingRunnerSet. The `Template` field is ephemeral runner template while the `ListenerField` is pod template spec for the listener
We don't have restrictions on the helm level the same way we don't have any restrictions for the ephemeral runner :relaxed: